src/App.js

```diff
@@ -1,13 +1,16 @@
+import React from 'react';
+// eslint-disable-next-line import/extensions
+import DoCalculation from './components/calculator';
 import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Math Magician</h1>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
+        <DoCalculation />
```

Comment:
- [ ] Currently running the app fails with the error: `Module not found: Can't resolve './components/calculator' in '/home/mike/Documents/Projects/Math-Magician/src'`, kindly fix it.